src/index.html

```diff
@@ -16,36 +16,40 @@
   </head>
   <body>
     <header>
-      <h1><span>fresh</span>Picks<img src="../images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
-      <form id="search" role="search">
-        <input id="search-input" type="text" autocomplete="off" placeholder="Search Recipes..."></input>
-        <label for="search-input" class="search-label">Search</label>
-        <button class="nav-btn search-btn" type="button" name="button"><img src="../images/search.png" alt="search button"></button>
+      <h1><span>fresh</span>Picks<img src="../images/apple-logo.png" alt="bright green apple icon" class="header-apple-icon"></h1>
+      <form tabindex ="0" id="search" role="search">
```

Comment:
tabindex? nice is that an Aria tag?

Reply:
it looks like it is! aria tags are really cool.

Reply:
So tabindex allows you to make an element focusable for keyboard users! It is an HTML attribute.

 The h1 might not need it, but it can also allow you to change the display order of elements on the page by adding tabindex ="1" or tabinex ="-1" and so on.